plenary/plenary-26.yaml

```diff
@@ -0,0 +1,211 @@
+    title: "Disbandment of liaison with LAS"
+    identifier: 297
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+        
+    actions:
+      - type: resolves
+        date_effective: 2009-09-25
+        message: ISO/TC 154 resolves to disband the liaison with LAS
+        
+        
+  - dates:
+      - 2009-09-25
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Category A liaison with UN/CEFACT"
+    identifier: 298
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+        
+    actions:
+      - type: resolves
+        date_effective: 2009-09-25
+        message: ISO/TC 154 resolves to establish a Category A liaison with UN/CEFACT
+        
+        
+  - dates:
+      - 2009-09-25
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Disbandment of liaison with UN"
+    identifier: 299
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+        
+    actions:
+      - type: resolves
+        date_effective: 2009-09-25
+        message: ISO/TC 154, considering the existing liaisons with UNECE, UNCTAD and UN/CEFACT, resolves to disband the liaison with UN
+        
+        
+  - dates:
+      - 2009-09-25
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Category A liaison with IATA"
+    identifier: 300
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+        
+    actions:
+      - type: resolves
+        date_effective: 2009-09-25
+        message: ISO/TC 154 resolves to establish a Category A liaison with IATA
+        
+        
+  - dates:
+      - 2009-09-25
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Liaison officer from ISO/TC 154 t o ISO/TC 204"
+    identifier: 301
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+        
+    actions:
+      - type: resolves
+        date_effective: 2009-09-25
+        message: ISO/TC 154 resolves to appoint Mr. Vankemmel as liaison officer from ISO/TC 154 to ISO/TC 204
+        
+        
+  - dates:
+      - 2009-09-25
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Appreciation"
+    identifier: 302
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: ACCLAMATION
+
+    actions:
+      - type: thanks
+        degree: unanimous
+        message: ISO/TC 154 expresses its sincere appreciation to Mr. Wakaizumi, the ISO/TC154 Japan Committee and JIPDEC for arranging the twenty-sixth meeting of ISO/TC 154 in Tokyo and hospitality
```

Comment:
Regarding this part, is it correct to put `thanks` as type of action?
Original source reference:
![capture4](https://user-images.githubusercontent.com/33627611/71758910-b756de00-2e7b-11ea-98a1-afaf1a58e591.JPG)

Reply:
Yes, `thanks` is good.